build/ci/templates/test-phase.yml

```
@@ -128,6 +128,13 @@ steps:
 
       arguments: '-m pip install --upgrade -r ./build/test-requirements.txt'
 
+  - task: CmdLine@1
+    displayName: 'pip install  functional requirements'
+    inputs:
+      filename: python
+      arguments: '-m pip install --upgrade -r ./build/functional-test-requirements.txt'
+    condition: variables['HasFunctionalRequirements']
```

You could do away with `HasFunctionalRequirements`, and do this instead:

```suggestion
    condition: eq(variables['TestSuiteName'], 'test:functional')
```

...I leave that to you - it isn't really much different either way.